apps/base-docs/docs/tools/data-indexers.md

```diff
@@ -84,6 +85,21 @@ To get started, visit the [documentation](https://docs.envio.dev/docs/overview)
 
 ---
 
+## Moralis
+
+[Moralis](https://moralis.io/?utm_source=base-docs&utm_medium=partner-docs) offers the most comprehensive data APIs for crypto, offering both indexed and real-time data across 15+ chains. Moralis' APIs include portfolio and wallet balances, NFT data, token data, price data, candlestick data, net worth data, and a lot more. All of our data is enriched with things like metadata, parsed events and address labels.   
```

Comment:
This will be included in Base docs rather than Moralis docs, so consider updating the wording:

`All of our data is enriched with things like metadata, parsed events and address labels.`

to

`All of the data is enriched with things like metadata, parsed events and address labels.`